### PR TITLE
Add Website branch

### PR DESCRIPTION
**Pull request by giladshanan**
Hi @wolfram74 and @allyjo, I'm all done with my first website.  Please take a look and let me know what you think.  Thanks!

### CODE REVIEW

File: feedback.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <option value="Constructive Criticism">Constructive Criticism</option>
+      <option value="Recipe Question">Recipe Question</option>
+    </select>
+  </div>
+
+  <div>
+    <label>Enter your message below:</label>
+    <br>
+    <textarea name="comments" rows="15" cols="70">
+      Start typing here...
```
Comment (wolfram74) [inline feedback.html]: Might look into the placeholder attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Soul Dumpling</title>
+</head>
+<body>
+<h1>Soul Dumpling: <br> Heart Warming Comfort Food Recipes from Around the World</h1>
```
Comment (wolfram74) [inline index.html]: If you're including a br tag, maybe split the line up a bit more.

File: recipes.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Recipes</title>
+</head>
+<body>
+  <p>
+    <h2>Recipes</h2> ...and tasty bites from <em>all around the world.</em>
```
Comment (wolfram74) [inline recipes.html]: since h2 is a block element, you might consider doing a new line for the extra text

File: recipes.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  </p>
+
+  <div>
+    <ul>
+      <li>
+        <a href= "http://www.seriouseats.com/recipes/2015/03/the-best-japanese-pork-and-cabbage-dumplings-gyoza-recipe.html">Japanese Pork and Cabbage Dumplings</a>
```
Comment (wolfram74) [inline recipes.html]: these a tags are all pretty long, having their visible content be on a separate line  from the href or opening tag might make editing later easier.

File: feedback.html (hunk 1 above)
Comment (giladshanan) [reply feedback.html]: Good idea, unfortunately it looks like the placeholder works with the input tag, not the textarea tag.

File: index.html (hunk 2 above)
Comment (giladshanan) [reply index.html]: Thanks, will do

File: recipes.html (hunk 3 above)
Comment (giladshanan) [reply recipes.html]: Good call

File: recipes.html (hunk 4 above)
Comment (giladshanan) [reply recipes.html]: 👍

File: feedback.html (hunk 1 above)
Comment (wolfram74) [reply feedback.html]: documentation is sometimes misleading, the compiler is the final arbiter of truth so think of the smallest/fastest way to try it out and just experiment https://jsfiddle.net/742Lv52m/

Comment (giladshanan) [reply feedback.html]: I actually did play around with it, when I tried adding a placeholder to the feedback.html page it didn't render in Chrome!  But now I'll try again, maybe I missed some punctuation somewhere.

Comment (giladshanan) [reply feedback.html]: I added a placeholder attribute but its not working for me in Chrome.  Not sure why but I'm adding the commit so you can take a look.

**Comment by giladshanan** [comment]
Hey @wolfram74 I think the placeholder should be working now.  I have 'fix placeholder' on my to do list, is there another instance that we had been talking about or is this the one?

**Pull request merged**